src/utils/account.js

```diff
@@ -0,0 +1,5 @@
+
+export const validateEmail = (email) => {
+    let re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
This is way too complicated check, it's very hard to determine by reading what would match it. I think it makes sense to just make much simpler check (e.g. check whether string contains two non-empty parts separated by @).

We'll send e-mail to do final validation anyway.

Reply:
Will add simple validation